include/flashinfer/page.cuh

```diff
@@ -16,6 +16,7 @@
 #ifndef FLASHINFER_PAGE_CUH_
 #define FLASHINFER_PAGE_CUH_
 
+#include <assert.h>
```

Comment:
Would you mind changing them to `FLASHINFER_CHECK`? (defined in https://github.com/flashinfer-ai/flashinfer/blob/341ae09ecd33c441de6060ef51c5f369ec020694/include/flashinfer/exception.h#L41)

assert would only work when you compile the program in debug mode, not release mode.